prover/src/domain.rs

```diff
@@ -81,6 +100,19 @@ impl<B: StarkField> StarkDomain<B> {
         self.ce_to_lde_blowup
     }
 
+    /// Returns (offset * g^(step))^degree_adjustment.
+    pub fn get_ce_x_power_at<A: Air<BaseField = B>>(
```

Comment:
We could try adding `#[inline(always)]` attribute to this function (and maybe some other functions). Sometimes, this improves performance.

Reply:
Sure, I did here and in the helper function on the same file.